scripts/flush-dev-db.sh

```diff
@@ -1,18 +1,49 @@
 #!/bin/bash
-
 set -euo pipefail
 
+function check_dep {
+  if ! command -v "${1}" &>/dev/null; then
+    echo "You need '${1}' - '${2}'"
+    exit 1
+  fi
+}
+
+# Check dependencies
+check_dep "psql" "brew install postgresql"
+check_dep "jq" "brew install jq"
+check_dep "aws" "brew install awscli"
+check_dep "kubectl" "asdf install kubectl 1.19.15"
+
+# check for the ENV variable
+set +u
+if [[ -z "${ENV}" ]]; then
+  echo "You must set the ENV variable - allowed values: [dev, preprod, prod]"
```

Comment:
prod seems pretty gnarly/dangerous. surely we cant/shouldnt have access to prod?

Reply:
Fair point - will restrict to dev or pre-prod 👍